lib/mix/lib/releases/assembler.ex

```diff
@@ -810,6 +810,9 @@ defmodule Mix.Releases.Assembler do
       {:mkdir, "releases/<%= release_version %>/commands"} |
       Enum.map(release.profile.commands, fn {name, path} ->
         {:copy, path, "releases/<%= release_version %>/commands/#{name}.sh"}
+      end) ++ #Hack. This allows for .bat stuff to be run on windows (instead of just forcing .sh)
```

Comment:
To remain backwards compatible though, we should give anything without an extension, the `.sh` extension. We can do a deprecation cycle later to always require one.